Hello Microverse Pull Request

dselasea opened the pull request:
## Hello Microverse
In this pull request I,
- Added stylelintrc package
- Added stylelintrc.json
- Used Microverse README.md template
- Updated style.css

dselasea committed:
updated README.md file

sajid-munawar reviewed (changes requested):
# Changes Required 
 ### Hi @dselasea ,
 Nice job, you are almost there.
 
 ![](https://camo.githubusercontent.com/34482800e1b92cb79ddca06edb4051edc407cb190f896e1e949e596aa9073851/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
 
 ## (Highlights) Good Points: 
 * Followed Github flow :heavy_check_mark:
 * Great PR Summary description :heavy_check_mark: 
 ## (Changes Required) Aspects to improve: 
 *  ### Please do the following:
- [ ] Please kindly add a more descriptive PR title.
 See a demo below:

https://user-images.githubusercontent.com/52934128/176308229-e31b2c8d-110b-4823-a2a8-31c1b7d7c73d.mp4

- [ ] I kindly suggest that you add a more descriptive PR title and description. 
## EXAMPLE
```
## Hello World
In this Pull Request, I:
- Created branches with different base branches.
- Followed GitHub flow.
- Added a descriptive and easy to understand README
- Added a HTML file
- Added a CSS File
- Added a h1 text of "Hello-World"
```


- [ ] Please refer to this article: please read it and apply it in your work from now on: [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md);


- [ ] Kindly ensure your linter results pass * AKA: Linter should report NO errors.
 ## Stlylint AutoFix
- [ ] if you have a lot of CSS line errors, I kindly suggest you fix them with `npx stylelint "**/*.{css,scss}" --fix`, but please get used to fixing the errors line by line also because it'll help you learn best practices.


- [ ] I suggest you kindly add a [README](https://github.com/microverseinc/readme-template) file to your repo, using the Microverse [template](https://github.com/microverseinc/readme-template) as this will give users detailed information about your project. Here is demo below

https://user-images.githubusercontent.com/52934128/176309876-98f49c4e-3e62-4edd-9bf3-51a66f48e2cd.mp4


 ## [Optional] suggestions:
 * Nothing to mention.
 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 **Cheers and Happy coding! :clap::clap::clap:**
 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag me in your question so I can receive the notification.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

dselasea committed:
modified style.css and added .stylelintrc.json

dselasea commented:
## Hello Microverse
In this pull request I,
- Added stylelintrc package
- Added stylelintrc.json
- Used Microverse README.md template
- Updated style.css

OyePriscilla commented on README.md:
```diff
@@ -0,0 +1,76 @@
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
```
- [ optional] As at now, you are the only author of this project. Kindly remove the second author and provide your details for the first author. 😄  Doing this will make your README file looks more professional.

OyePriscilla commented on README.md:
```diff
@@ -0,0 +1,76 @@
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
```
- [ optional] Kindly list the steps that a developer needs to take to access a local copy of your project. 😄

OyePriscilla reviewed (changes requested):
Hi @dselasea  😃 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![so-close](https://user-images.githubusercontent.com/103386183/196423863-0f522555-1b03-455e-a452-b822cdcfc398.gif)

## Highlights
Good commits messages ✔️ 
GitHub flow observed ✔️ 
Linters checks passed ✔️ 

## Required Changes ♻️

- [ ] Kindly add a descriptive PR title and summary as the previous reviewer suggested. Kindly click on the edit icon and look through this [Demo](https://user-images.githubusercontent.com/52934128/177452738-0610c008-a53a-454e-8aac-635cb4ed812c.gif)

## Your PR title and summary

![A9](https://user-images.githubusercontent.com/103386183/199704152-d5a3e294-665f-4b67-91d7-22b1e87c3eb6.png)
://user-images.githubusercontent.com/103386183/199703988-951b8b4d-82b8-44fc-8eb7-32392931b9a3.png)


## Optional suggestions

- [optional]  For subsequent PR, kindly ensure that your branch name speaks to the activity you're doing. I could see `testbranch `for this PR. This is not so right, it can be confusing.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

dselasea committed:
updated the README.md file

BregornOriginal reviewed (approved):
Hi @dselasea ,
# Your project is complete!  There is nothing else to say other than... it's time to merge it 🎉🎉🎉 

# Congratulations! 🥳 🥳 🥳 

## Highlights

✅ Linters are passing
✅ Good PR description
✅ Good PR naming
✅ README looks professional

![giphy](https://user-images.githubusercontent.com/93630700/199723543-926a77a8-f441-49c9-9a80-434d798ce666.gif)

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

dselasea commented:
- Pull Request Approved by the code reviewer
